src/ResourceManager/KeyVault/Commands.KeyVault.Test/ScenarioTests/KeyVaultManagementController.cs

```diff
@@ -38,10 +40,11 @@ namespace Microsoft.Azure.Commands.KeyVault.Test
     public class KeyVaultManagementController
     {
 
-        private CSMTestEnvironmentFactory csmTestFactory;
+        private LegacyTest.CSMTestEnvironmentFactory csmTestFactory;
```

Comment:
If this is legacy now, there should be a newer way of testing. Can we use that instead?